app/mailers/course_create_email_mailer.rb

```diff
@@ -22,6 +24,6 @@ def create_course_url(course)
   end
 
   def format_create_datetime(datetime)
-    datetime.strftime("%-k:%M%P on %-e %B %Y")
+    datetime.strftime("%-k:%M on %-e %B %Y")
```

Comment:
I remember working on a similar issue https://github.com/DFE-Digital/teacher-training-api/pull/1237.

TBH, I think we should just have a standard global date/time format so that we do not have to think about date/time format every time we need to playback date/time to users.

Reply:
Actually looking back at that PR and the ticket I was the one that changed this formating from what it was and this was after a discussion between Paul G, Sam C and myself. I might still have the slack conversation, I'll see if I can find it.

Reply:
Formatting fixed